### DISCUSSION TITLE
添加中英切换按键

### DISCUSSION THREAD

**Discussion by imfuxiao** [imfuxiao/Hamster#61] [Show and tell]
添加中英文切换按键. 同时在空格显示当前中英状态.

这个需求有个情况需要讨论下.

中文模式下, 在已经有候选字的情况下, 切换英文模式. 
1. 清空中文输入, 开始英文输入
2. 候选栏字母上屏, 并开启英文输入
3. 首选候选字上屏, 并开启英文输入

应该只有这3种形式了, 应该用哪种处理方式呢? 目前使用的 1 方案. 如果有其他想法可以在此讨论.

**Comment by ctermiii** [comment]
个人习惯方式2

**Reply by tingting-d** [reply]
习惯方式2，目前大多数输入法都是采用的2

**Comment by Attiv** [comment]
设置里加个选项。。123
